staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal.go

```diff
@@ -45,10 +45,31 @@ const (
 	ILBFinalizerV1 = "gke.networking.io/l4-ilb-v1"
 	// ILBFinalizerV2 is the finalizer used by newer controllers that implement Internal LoadBalancer services.
 	ILBFinalizerV2 = "gke.networking.io/l4-ilb-v2"
+	//ServicePrefix is used to prefix annotations on the service
+	ServicePrefix = "service.kubernetes.io"
+	// TCPForwardingRuleKey is the key to record the GCP TCP Forwarding Rule Name
+	TCPForwardingRuleKey = ServicePrefix + "/tcp-forwarding-rule"
+	// UDPForwardingRuleKey is the key to record the GCP UDP Forwarding Rule Name
+	UDPForwardingRuleKey = ServicePrefix + "/udp-forwarding-rule"
+	// BackendServiceKey is the key to record the GCP backend service name
+	BackendServiceKey = ServicePrefix + "/backend-service"
+	// FirewallRuleKey is the key to record the GCP firewall rule name
+	FirewallRuleKey = ServicePrefix + "/firewall-rule"
```

Comment:
Can you add another key called 'FirewallRuleForHcKey = ServicePrefix + "/firewall-rule-for-hc"' and set the value as fwHCName in line 525?

Reply:
Good catch. New version should have this annotation as well.